extension/src/experiments/model/util.ts

```diff
@@ -13,8 +13,9 @@ export const getDataFromColumnPath = (
     columnPath === 'Created' && collectedVal
       ? formatDate(collectedVal)
       : collectedVal?.value || collectedVal
+
   return {
     splitUpPath,
-    value: value || null
+    value: value === 0 || value ? value : null
```

Comment:
Falsy values are `undefined`, `null`, `NaN`, `0`, `""`, and `false`. With this change, we will show the column if it has a value that is 0 or it's none falsy value.

Reply:
Yes, it is possible to have boolean values. Might be best to just show everything so:
```
return {
  splitUpPath,
  value,
}
```

Reply:
I think `undefined` would still be useful to see. If it's not set for a row and it is in all others, it's actually what makes it jump out. Same for the empty string (for that one, I think I'd replace it with a literal `empty` just to make it clear.

Reply:
Makes sense! Updated the code to hopefully properly stringify any possible values!